### DISCUSSION TITLE
Decimal separator

### DISCUSSION THREAD

**Discussion by yangetze** [Megabit/Blazorise#3065] [Q&A] [answered]
Hey, I got two questions about the numericedit component. 

1.  I need to show a decimal type with 4 places, even if the places are zero. Like 1,000. 
![imagen](https://user-images.githubusercontent.com/23171333/139346661-bdc5ac19-a560-4715-84f4-12bae36f7cd9.png)
![imagen](https://user-images.githubusercontent.com/23171333/139347091-9e4cf34d-5179-4e7d-be06-7bf0be1f0852.png)

2. I need that the decimal separator it'll be comma. But when I try, I don't get any result 

This is my code
```
<NumericEdit TValue="decimal" @bind-Value="@NewCotizacion.TasaDeCambio" Disabled="true" Plaintext="true"
                                                        Decimals="4" DecimalsSeparator=","></NumericEdit>
```

I read about Culture atribbute, but I don't know how to use it, and I read about Browser problems. But I don't know how to advance with this. What recommend me?

I'm using the control only for show. Is there any better practice?

Version: 0.9.4.6

**Comment by stsrki** [comment, marked as answer]
There are no better options for now, unfortunately. Can't do much on 0.9.4.x as that would be a breaking change. Eventually, we can do it on the next milestone, since we have almost done with the 0.9.5 release and not much time is left for any major changes. We do have plans to add support for advanced numeric input that is based on the excellent library `autoNumeric`, #2983. But as mentioned, it will be in the next milestone only.

**Reply by yangetze** [reply]
Okkkk. Thnks!!!
